src/docs/guides/luminus.md

```diff
@@ -0,0 +1,201 @@
+
+Open your `project.clj` file and ensure you have the following under `:dependencies`:
+
+```clojure
+...
+[cheshire "5.10.0"]
+```
+
+Run the command below in your terminal to ensure it is installed:
+
+```bash
+lein deps
+```
+
+2. Create a `nixpacks.toml` file in the root directory of the app. 
+
+The [nixpacks.toml file](https://nixpacks.com/docs/configuration/file) is a configuration file used by Nixpacks, a build system developed and used by Railway, to set up and deploy applications. 
+
+In this file, you can specify the instructions for various build and deployment phases, along with environment variables and package dependencies.
+
+Add the following content to the file:
+
+```toml
+# nixpacks.toml
+
+[start]
+cmd = "java -jar $(find ./target -name '*.jar' ! -name '*SNAPSHOT*') migrate && java -jar $(find ./target -name '*.jar' ! -name '*SNAPSHOT*')"
+
+```
+
+Here, we are specifically instructing Nixpacks to use the following command to start the app.
+
+The command searches for all `.jar` files in the `target` directory (where the standalone JAR file is located after the build), excludes any file with "SNAPSHOT" in its name, and passes the selected file to `java -jar` to run.
+
+It starts by running the JAR file with the `migrate` option to apply database migrations. Once migrations are complete, it reruns the JAR file to launch the application.
+
+## Deploy the Luminus App to Railway
+
+Railway offers multiple ways to deploy your Clojure app, depending on your setup and preference. 
+
+### One-Click Deploy from a Template
+
+If you’re looking for the fastest way to get started, the one-click deploy option is ideal.
+
+Click the button below to begin:
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/DsDYI2)
+
+We highly recommend that [you eject from the template after deployment](/guides/deploy#eject-from-template-repository) to create a copy of the repo on your GitHub account.
+
+**Note:** You can also choose from a <a href="https://railway.app/templates?q=clojure" target="_blank">variety of Clojure app templates</a> created by the community.
+
+### Deploy from the CLI
+
+1. **Install the Railway CLI**:
+    - <a href="/guides/cli#installing-the-cli" target="_blank">Install the CLI</a> and <a href="/guides/cli#authenticating-with-the-cli" target="_blank">authenticate it</a> using your Railway account.
+2. **Initialize a Railway Project**:
+    - Run the command below in your Luminus app directory. 
+        ```bash
+        railway init
+        ```
+    - Follow the prompts to name your project.
+    - After the project is created, click the provided link to view it in your browser.
+3. **Add a Postgres Database Service**:
+    - Run `railway add -d postgres`.
+    - Hit **Enter** to add it to your project.
+    - A database service will be added to your Railway project.
+4. **Add a Service and Environment Variable**:
```

Comment:
I can't get over how much better this new feature makes the cli guides

Reply:
It's amazing. Thank you @Milo123459